<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmoothfitWidget</class>
 <widget class="QWidget" name="SmoothfitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>616</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>135</height>
      </size>
     </property>
     <property name="features">
      <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
     </property>
     <property name="allowedAreas">
      <set>Qt::AllDockWidgetAreas</set>
     </property>
     <property name="windowTitle">
      <string>Fitting Steps</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>228</width>
            <height>588</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QToolBox" name="toolBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolBox::tab {
         background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                     stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
         border-radius: 5px;
         color: black;
     }

     QToolBox::tab:selected { /* italicize selected tabs */
         font: bold;
         color: black;
     }
QToolBox {
    padding : 0
}</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="tabSpacing">
              <number>2</number>
             </property>
             <widget class="QWidget" name="alignPage">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>224</width>
                <height>468</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <attribute name="label">
               <string>1. Align</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
               <item>
                <widget class="QGroupBox" name="alignSettingsGroupBox">
                 <property name="title">
                  <string>Settings</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="alignLoadSaveWidgets" native="true">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <layout class="QGridLayout" name="gridLayout" rowstretch="2">
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <property name="spacing">
                      <number>3</number>
                     </property>
                     <item row="0" column="1">
                      <widget class="QPushButton" name="alignSaveButton">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="text">
                        <string>Save</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QPushButton" name="alignLoadButton">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="text">
                        <string>Load</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="alignScaleWidgets" native="true">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                     <property name="horizontalSpacing">
                      <number>3</number>
                     </property>
                     <property name="verticalSpacing">
                      <number>3</number>
                     </property>
                     <property name="margin">
                      <number>3</number>
                     </property>
                     <item row="2" column="0">
                      <widget class="QLabel" name="alignScaleLabel">
                       <property name="text">
                        <string>Scale:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QLineEdit" name="alignScaleLineEdit"/>
                     </item>
                     <item row="3" column="0">
                      <widget class="QLabel" name="alignRotationLabel">
                       <property name="text">
                        <string>Rotation:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="1">
                      <widget class="QLineEdit" name="alignRotationLineEdit"/>
                     </item>
                     <item row="4" column="0">
                      <widget class="QLabel" name="alignOffsetLabel">
                       <property name="text">
                        <string>Offset:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="1">
                      <widget class="QLineEdit" name="alignOffsetLineEdit"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="alignResetButton">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="alignAutoCentreButton">
                 <property name="text">
                  <string>Auto Centre</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="projectPage">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>224</width>
                <height>468</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <attribute name="label">
               <string>2. Project</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="projectClearButton">
                 <property name="text">
                  <string>Clear Projections</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="projectPointsButton">
                 <property name="text">
                  <string>Project Points</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="fitPage">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>224</width>
                <height>468</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <attribute name="label">
               <string>3. Fit</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
               <item>
                <widget class="QGroupBox" name="fitSettingsGroupBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="title">
                  <string>Settings</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="fitSettingsLoadSaveWidgets" native="true">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2" rowstretch="2">
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <property name="spacing">
                      <number>3</number>
                     </property>
                     <item row="0" column="0">
                      <widget class="QPushButton" name="fitLoadButton">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="text">
                        <string>Load</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QPushButton" name="fitSaveButton">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="text">
                        <string>Save</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="fitSettingsWidget" native="true">
                    <layout class="QFormLayout" name="formLayout_2">
                     <property name="fieldGrowthPolicy">
                      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                     </property>
                     <property name="verticalSpacing">
                      <number>3</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>3</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>3</number>
                     </property>
                     <item row="0" column="0">
                      <widget class="QLabel" name="fitStrainPenaltyLabel">
                       <property name="text">
                        <string>Strain Penalty:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="fitStrainPenaltyLineEdit"/>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="fitEdgeDiscontinuityPenaltyLabel">
                       <property name="text">
                        <string>Edge Discontinuity Pen.:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLineEdit" name="fitEdgeDiscontinuityPenaltyLineEdit"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="fitPerformButton">
                 <property name="text">
                  <string>Perform Fit</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <widget class="QPushButton" name="viewAllButton">
                <property name="text">
                 <string>View All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="doneButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Done</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="AlignmentSceneviewerWidget" name="sceneviewerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AlignmentSceneviewerWidget</class>
   <extends>QWidget</extends>
   <header>alignmentsceneviewerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
